gcloud/bigtable/happybase/table.py

```diff
@@ -561,3 +565,135 @@ def _gc_rule_to_dict(gc_rule):
                 if key1 != key2:
                     result = {key1: rule1[key1], key2: rule2[key2]}
     return result
+
+
+def _next_char(str_val, index):
+    """Gets the next character based on a position in a string.
+
+    :type str_val: str
+    :param str_val: A string containing the character to update.
+
+    :type index: int
+    :param index: An integer index in ``str_val``.
+
+    :rtype: str
+    :returns: The next character after the character at ``index``
+              in ``str_val``.
+    """
+    if six.PY3:  # pragma: NO COVER
```

Comment:
Yay tox. Hmm. I feel like if this is a common theme it should be in `.coveragerc`, not in the code.